### ISSUE TITLE
Android Demo doesn't work

### ISSUE THREAD

**Issue by jkomyno** [FormidableLabs/victory-native#114]
This isn't a copy of #68. I've been trying to run both this project's demo and [this separate one](https://github.com/FormidableLabs/victory-native-demo) with no success. Here's the error I'm getting.
Please note that I've console.log'd every single component imported in demo.js, and none of them is undefined.

![](http://i.imgur.com/sQo5z9o.png)

I'm running Windows 10 on my PC, react-native-cli is 2.0.1 and react-native is the same exact version as the required dependencies in each package.json.

**Comment by chrisbolin** [comment]
@jkomyno i'm sorry about the errors! let's concentrate on getting https://github.com/FormidableLabs/victory-native-demo to work, as that's the one we've tested the most with Android. It looks like that is what your screenshot is from, correct?

I haven't done is test on Windows + Android; I'm hoping this isn't an issue there.

Could you try using https://github.com/FormidableLabs/victory-native-demo as a base and just render a simple `react-native-svg` test? Here's one from their [readme](https://github.com/react-native-community/react-native-svg#usage):

```jsx
import Svg,{
    Circle,
    Ellipse,
    G,
    LinearGradient,
    RadialGradient,
    Line,
    Path,
    Polygon,
    Polyline,
    Rect,
    Symbol,
    Text,
    Use,
    Defs,
    Stop
} from 'react-native-svg';

class SvgExample extends Component {
    render() {
        return (
            <Svg
                height="100"
                width="100"
            >
                <Circle
                    cx="50"
                    cy="50"
                    r="45"
                    stroke="blue"
                    strokeWidth="2.5"
                    fill="green"
                />
                <Rect
                    x="15"
                    y="15"
                    width="70"
                    height="70"
                    stroke="red"
                    strokeWidth="2"
                    fill="yellow"
                />
            </Svg>
        );
    }
}```

**Comment by chrisbolin** [comment]
Note that Expo includes their own version of react-native-svg, so you won't need to install anything else.

**Comment by chrisbolin** [comment]
If this doesn't work, could you verify that a basic https://github.com/react-community/create-react-native-app application works on your system?

**Comment by jkomyno** [comment]
My understanding is that VictoryArea is broken, since I can render every other component without issues. If I comment it out, the example works well.

**Comment by jkomyno** [comment]
I have no issues with react-native-svg or with the other components imported by victory-native, at all.

**Comment by chrisbolin** [comment]
Ah! you hit https://github.com/FormidableLabs/victory-native/issues/115 then. That is something I was never able to reproduce locally, but a number of users hit it. they were able to fix it with `master`, which was released as `victory-native@0.13.0`

**Comment by jkomyno** [comment]
Good to know, thanks! As of now I have @0.12.0 installed in an app I'm working on at the office, so I will upgrade and see if VictoryArea works as expected!
I'll let you know in some hours, now I have lunch break

**Comment by chrisbolin** [comment]
sounds good! with these spooky react-native bugs, i always recommend a fresh install of all dependencies (especially if a simple upgrade doesn't work the first time)

**Comment by jkomyno** [comment]
It works: I can finally use VictoryArea both in the demo and in my app!
Thanks for the update.

**Comment by chrisbolin** [comment]
thanks for the update @jkomyno! This has been helpful for me -- I know see the underlying issue was conflicting deps. Will fix the demo repo.

**Comment by jkomyno** [comment]
@chrisbolin I have just noted that after upgrading to victory-native@0.13.0 the VictoryPie layout seems broken. Related issue: #125

**Comment by chrisbolin** [comment]
@jkomyno thanks! i'll make sure the demo depends on a "good" version of victory-native

**Comment by boygirl** [comment]
@jkomyno that was a release timing issue on our part. The VictoryPie layout is fixed in version 0.14.0
